test/integration/cloudevents/statusfeedback_test.go

```diff
@@ -541,4 +541,186 @@ var _ = ginkgo.Describe("ManifestWork Status Feedback", func() {
 			}, eventuallyTimeout, eventuallyInterval).ShouldNot(gomega.HaveOccurred())
 		})
 	})
+
+	ginkgo.Context("DaemonSet Status feedback", func() {
```

Comment:
@skeeey why do we still need to create separated integration test case for cloudevent?

Reply:
the could events supports single manifest, for single manifest, some integration cannot pass